platform_umbrella/apps/home_base_web/lib/home_base_web/controllers/installation_script_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule HomeBaseWeb.InstallScriptController do
+  use HomeBaseWeb, :controller
+  use HomeBaseWeb, :verified_routes
+
+  require EEx
+
+  EEx.function_from_file(:defp, :render_install_script, "priv/raw_files/install_script.sh", [:spec_url, :version])
+
+  action_fallback HomeBaseWeb.FallbackController
+
+  def show(conn, %{"installation_id" => install_id}) do
+    script =
+      render_install_script(
+        url(conn, ~p'/api/v1/installations/#{install_id}/spec'),
+        CommonCore.Version.version()
```

Comment:
This is currently 0.19.0 which doesn't actually have a `bi` release yet. Not sure how we want to handle that...

Reply:
Let's have a `CommonCore.Defaults`  for `stable_bi_version` or something similar so that we can explicitly say which bi release to use.